changelog/_unreleased/2021-02-18-add-retryable-transaction-to-prevent-deadlocks.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Adds the RetryableTransaction to prevent database deadlocks
+issue:
+---
+# Core
```

Comment:
Sorry I saw it only now. 

Since your change means that plugins have to adapt their source code, please maintain the `UPGRADE INFORMATION` part of the changelog file.

Here is just an example what people have to look for and how it has to look before and after.